### PR TITLE
Align with VSCodeVim's window navigation

### PR DESCRIPTION
**Pull request by HyunggyuJang**
Currently, focus left and focus right doesn't respect "left" and "right" directions. It just goes anticlockwise and clockwise navigation.

With this change, now it aligns well with expected behavior, left and right wise window movement.

### CODE REVIEW

**Comment by stevenguh** [comment]
This is the commit I found when this is added:
https://github.com/VSpaceCode/VSpaceCode/commit/4e30245d2c01d495a20659a898e1369ba093cd51

I have no idea why I didn't use this comment indeed. This PR seems to work fine for me.  @MarcoIeni Any thoughts?

File: CHANGELOG.md
Comment (MarcoIeni) [inline CHANGELOG.md:18]:
```suggestion
-   `SPC w` + `h`, `j`, `k`, `l` navigate to the expected direction.
```

**Review by MarcoIeni — APPROVED** [review]
Perfect, thanks for this! Just changed the changelog :)
Steven, merge this if you like the changelog entry 👍

**Comment by stevenguh** [comment]
Thanks @MarcoIeni and @HyunggyuJang!! Will probably cut another small release once we have PR merged for #316

**Pull request merged**
